### PR TITLE
feat: Add support for signMessage in Sender

### PR DESCRIPTION
**Pull request by kujtimprenkuSQA**
# Description

This PR adds support for `signMessage` to Sender.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# Type of change.

- [ ] FIX - a PR of this type patches a bug.
- [x] FEATURE - a PR of this type introduces a new feature.
- [ ] BUILD - a PR of this type introduces build changes.
- [ ] CI - a PR of this type introduces CI changes.
- [ ] DOCS - a PR of this type introduces DOCS improvement.
- [ ] STYLE - a PR of this type introduces style changes.
- [ ] REFACTOR - a PR of this type introduces refactoring.
- [ ] PERFORMANCE - a PR of this type introduces performance changes.
- [ ] TEST - a PR of this type adds more tests.
- [ ] CHORE - a PR introduces other changes than the specified above.

### CODE REVIEW

File: examples/react/components/Content.tsx
Comment (kujtimprenkuSQA) [inline examples/react/components/Content.tsx:254]: Looks like the `message` is not being signed correctly, I added this check here just for testing and will be removed before merging:

image

Comment (kujtimprenkuSQA) [reply examples/react/components/Content.tsx:254]: Sender has fixed this issue, we're just waiting for the release on Chrome.

Comment (kujtimprenkuSQA) [reply examples/react/components/Content.tsx]: Marking this as resolved after testing out the sender locally, we now are just waiting for Sender to be published.

**Review by erditkurteshiSQA — APPROVED** [review]

**Pull request merged**
